ecommerce/enterprise/utils.py

```diff
@@ -529,42 +529,29 @@ def has_enterprise_offer(basket):
     return False
 
 
-def get_enterprise_catalog(site, enterprise_catalog, limit, page, endpoint_request_url=None):
+def get_enterprise_catalog(site, enterprise_catalog, page_size, page, endpoint_request_url=None):
     """
     Get the EnterpriseCatalog for a given catalog uuid.
 
     Args:
         site (Site): The site which is handling the current request
         enterprise_catalog (str): The uuid of the Enterprise Catalog
-        limit (int): The number of results to return per page.
+        page_size (int): The number of results to return per page.
         page (int): The page number to fetch.
         endpoint_request_url (str): This is used to replace the lms url with ecommerce url
 
     Returns:
         dict: The result set containing the content objects associated with the Enterprise Catalog.
         NoneType: Return None if no catalog with that uuid is found.
-
-    Response will look like
```

Comment:
Could we leave an example of the response in the docstrings of course with the actual data?

Reply:
I think about it but the response with actual data takes a lot of lines

Reply:
That should not stop you from doing it

Reply:
ok, in that case i will add